cli/cmd/vulnerabilty_exceptions.go

```diff
@@ -0,0 +1,354 @@
+//
+// Author:: Darren Murray(<darren.murray@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/AlecAivazis/survey/v2/core"
+	"github.com/lacework/go-sdk/api"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// vulnerability-exceptions command is used to manage lacework vulnerability exceptions
+	vulnerabilityExceptionCommand = &cobra.Command{
+		Use:     "vulnerability-exception",
+		Aliases: []string{"vulnerability-exceptions", "ve", "vuln-exception", "vuln-exceptions"},
+		Short:   "Manage vulnerability exceptions",
+		Long:    "Manage vulnerability exceptions to control and customize your alert profile for hosts and containers.",
+	}
+
+	// list command is used to list all lacework vulnerability exceptions
+	vulnerabilityExceptionListCommand = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all vulnerability exceptions",
+		Long:    "List all vulnerability exceptions configured in your Lacework account.",
+		Args:    cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			vulnerabilityExceptions, err := cli.LwApi.V2.VulnerabilityExceptions.List()
+			if err != nil {
+				return errors.Wrap(err, "unable to get vulnerability exceptions")
+			}
+			if len(vulnerabilityExceptions.Data) == 0 {
+				msg := `There are no vulnerability exceptions configured in your account.
+
+Get started by integrating your vulnerability exceptions to manage alerting using the command:
+
+    lacework vulnerability-exception create
+
+If you prefer to configure vulnerability exceptions via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Vulnerabilities > Exceptions.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(vulnerabilityExceptions)
+			}
+
+			var rows [][]string
+			for _, vuln := range vulnerabilityExceptions.Data {
+				rows = append(rows, []string{vuln.Guid, vuln.ExceptionName, vuln.ExceptionType, vuln.Status()})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "TYPE", "STATE"}, rows))
```

Comment:
This tables looks better (as we talked about):
```suggestion
			cli.OutputHuman(renderCustomTable([]string{"GUID", "NAME", "TYPE", "STATE"}, rows,
				tableFunc(func(t *tablewriter.Table) {
					t.SetBorder(false)
					t.SetAutoWrapText(false)
				})))
```